pipeline-model-api/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/ast/ModelASTStage.java

```diff
@@ -21,6 +21,7 @@
     private ModelASTWhen when;
     private ModelASTTools tools;
     private ModelASTEnvironment environment;
+    private Boolean failFast;
```

Comment:
I can bet a beer on that someone will try it and then complain that it doesn't work :)